installer/packer/scripts/admiral/configure_admiral.sh

```diff
@@ -25,6 +25,7 @@ fi
 
 data_dir=/data/admiral
 conf_dir=/etc/vmware/admiral
+javadir=/var/opt/OpenJDK-1.8.0.112-bin/bin/
```

Comment:
How about: `javadir=/var/opt/OpenJDK-*-bin/bin/`
Then you don't have to change this line when the dep is updated.